src/cmd_handlers.c

```diff
@@ -1011,6 +1016,48 @@ goto_cmd(const cmd_info_t *cmd_info)
 	return 0;
 }
 
+/* Opens external editor to edit specified register contents. */
+static int
+regedit_cmd(const cmd_info_t *cmd_info)
+{
+	const int regname = cmd_info->argc > 0 ? tolower(cmd_info->argv[0][0]) : DEFAULT_REG_NAME;
+	if(regname == BLACKHOLE_REG_NAME)
+	{
+		ui_sb_err("Cannot modify blackhole register.");
+		return CMDS_ERR_CUSTOM;
+	}
+
+	reg_t *reg = regs_find(regname);
+	if(reg == NULL)
+	{
+		ui_sb_err("Trying to edit uninitialized register.");
```

Comment:
`regs_find()` returns `NULL` for registers that don't exist at all, rather than uninitialized ones, which would imply that they can be somehow initialized and then used.